stable/prometheus-rabbitmq-exporter/templates/monitor.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       app: {{ template "prometheus-rabbitmq-exporter.name" . }}
       release: {{ .Release.Name }}
   endpoints:
-    - port: publish
+    - port: {{ .Values.service.externalPort }}
```

Comment:
You should be able to use the service's named port `rabbitmq-exporter` instead of hardcoding to the port number.

Reply:
Then it shoud be "rabbitmq-exporter" and not publish
This is what the service declares:
    - port: {{ .Values.service.externalPort }}
      targetPort: publish
      protocol: TCP
      name: rabbitmq-exporter

Reply:
Isn't that what I said? Use `rabbitmq-exporter`.

Reply:
I have updated it